### ISSUE TITLE
AspNetIdentity Sample Doesn't Authtenticate Created Users

### ISSUE THREAD

**Issue by farzad99** [IdentityServer/IdentityServer3.Samples#211]
I ran "AspNetIdentity" sample and created a number of users. Next I ran the "MVC FormPost Client" in the "Clients" sample and clicked the Signin link. In the login form I tried the login for the users I had created but they didn't work. Then I tried user name "bob" and password "bob" and it worked. Chrome remembered this user name and password from running other samples. Could somebody explain to me how come the created users don't work but userName "bob" with password "bob" works. In the page it even show's bob's email address as BobSmith@email.com as part of claims.

**Comment by brockallen** [comment]
You must have been pointing at the wrong server, or running the wrong one. Recall if you open multiple VS instances with the same IIS Express port settings, the last one wins.

**Comment by farzad99** [comment]
Thanks Brock for your reply. 

Just to double check I started the AspNetIdentity sample and looked at the IIS Express icon. It displayed the "webHost site" was running and it had the following urls:

http://localhost:12406/core/
https://localhost:44333/core/
http://localhost:12406/
https://localhost:44333/

They are the ports for the AspNetIdentity sample. Next I ran just the "MVC FormPost Client" in the "Clients" sample. IIS Express showed another site "MVC FormPost Client site" at the following url:

http://localhost:11716/

I clicked the SignIn link which took me to the login page at:

https://localhost:44333/core/login?signin=1d3fd27eef352dd13690ff52090a5427

I tried the logins for the users I had created and they all failed. I tried the userName "bob" and password "bob" which isn't in the database and it was successful. Then I clicked the Logout link which took me to the page at the following url:

https://localhost:44333/core/logout?id=651b86f588f1f4756e4a8ecdfc354702

I looked at the log file "c:\logs\IdentityServer3.AspNetIdentity.log". Here's what it says:

iisexpress.exe Information: 0 : [Owin.IdentityManagerAppBuilderExtensions]: 12/24/2015 10:14:25 PM -- Starting IdentityManager configuration
iisexpress.exe Warning: 0 : 2015-12-24 17:14:26.484 -05:00 [Warning] AuthorizationCodeStore not configured - falling back to InMemory
iisexpress.exe Warning: 0 : 2015-12-24 17:14:26.501 -05:00 [Warning] TokenHandleStore not configured - falling back to InMemory
iisexpress.exe Warning: 0 : 2015-12-24 17:14:26.501 -05:00 [Warning] ConsentStore not configured - falling back to InMemory
iisexpress.exe Warning: 0 : 2015-12-24 17:14:26.501 -05:00 [Warning] RefreshTokenStore not configured - falling back to InMemory

I don't think the log file warnings have to do with the issue that I'm experiencing. 

In summary created users don't work but the user name "bob" password "bob" which is definitely not in the database works.

**Comment by farzad99** [comment]
My samples zip file wasn't the latest (a few weeks old). I downloaded the latest samples zip file and that worked.
